data_collection/sns_scrape.py

```diff
@@ -0,0 +1,33 @@
+# We are going to scrape Twitter using snscrape
+
+import pandas as pd
+import numpy as np
+import snscrape.modules.twitter as sntwitter
+import csv
+import datetime
```

Comment:
There are some libraries that you've imported that have not been used in this script, those are `numpy` and `csv`. Our lint tool will show you some of this.
```suggestion
import pandas as pd
import snscrape.modules.twitter as sntwitter
import datetime
```